README.md

```diff
@@ -19,3 +28,13 @@ service and then testing it with the testkit.
 ## License
 
 Licensed under the Apache License (Version 2.0). See [LICENSE](LICENSE) for details.
+
+[travis-image]: https://travis-ci.org/exonum/exonum-testkit.svg?branch=master
+[travis-url]: https://travis-ci.org/exonum/exonum-testkit
+[appveyor-image]: https://ci.appveyor.com/api/projects/status/lhu2aykh9nyti78p/branch/master?svg=true
+[appveyor-url]: https://ci.appveyor.com/project/alekseysidorov/exonum-testkit/branch/master
+[gitter-image]: https://img.shields.io/gitter/room/exonum/exonum.svg?style=flat-square
```

Comment:
Nitpick: `?style=flat-square` sets the gitter badge apart from others 😃 You could remove `style`, or use badges with the same style from `shields.io`.